### PR TITLE
My branch

### PR DESCRIPTION
**Pull request by HanumanJat8698**
Pull request for assignment 2 is created.

### CODE REVIEW

File: Sorting_Array.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
+declare -a MYARRAY
+MYARRAY=( "$@" )
```
Comment (dharmeshpurohit2403) [inline Sorting_Array.sh:2]: what this command is doing?

File: Sorting_Array.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	do
+		INTVAR=`expr $EXTVAR + 1`
+		while [ $INTVAR -lt $# ]
+			do
+				if [ ${MYARRAY[$EXTVAR]} -gt ${MYARRAY[$INTVAR]} ]
+					then
+						#swapping
+						TEMP=${MYARRAY[$INTVAR]}
+						MYARRAY[$INTVAR]=${MYARRAY[$EXTVAR]}
+						MYARRAY[$EXTVAR]=$TEMP
```
Comment (dharmeshpurohit2403) [inline Sorting_Array.sh:28]: can we swap without using third variable?

File: Sorting_Array.sh (hunk 1 above)
Comment (HanumanJat8698) [reply Sorting_Array.sh:2]: Line 1: Declaring an array MYARRAY
Line 2: Storing all values passed using command line argument to MYARRAY.

File: Sorting_Array.sh (hunk 2 above)
Comment (HanumanJat8698) [reply Sorting_Array.sh:28]: Yes, it can be swapped by using + and ^(XOR) operator too.

**Pull request closed without merging**